openhab-scripting.gemspec

```diff
@@ -18,6 +18,7 @@ Gem::Specification.new do |spec|
   spec.metadata['homepage_uri'] = spec.homepage
   spec.metadata['source_code_uri'] = 'https://github.com/boc-tothefuture/openhab-jruby'
   spec.metadata['changelog_uri'] = 'https://github.com/boc-tothefuture/openhab-jruby/blob/main/CHANGELOG.md'
+  spec.metadata['rubygems_mfa_required'] = 'true'
```

Comment:
This might be problematic for automatic publishing of the gem via GitHub actions.

Reply:
> This might be problematic for automatic publishing of the gem via GitHub actions.

I agree. I have removed it and disabled that rubocop rule.